google-datacatalog-qlik-connector/src/google/datacatalog_connectors/qlik/prepare/constants.py

```diff
@@ -16,9 +16,12 @@
 
 ENTRY_ID_LENGTH = 64
 ENTRY_ID_PREFIX = 'qlik_'
-ENTRY_ID_STREAM = 'st_'
+ENTRY_ID_APP = 'app_'
+ENTRY_ID_STREAM = 'str_'
 NO_PREFIX_ENTRY_ID_LENGTH = ENTRY_ID_LENGTH - len(ENTRY_ID_PREFIX)
 
+TAG_TEMPLATE_ID_APP = 'qlik_app_metadata'
```

Comment:
And just to understand the naming convention being used in the constant names.

These examples:
`ENTRY_ID_APP`
`TAG_TEMPLATE_ID_APP`  

Do you think it makes to improve the readability by changing to ?
`ENTRY_ID_APP` => `ENTRY_APP_ID` 
`TAG_TEMPLATE_ID_APP`  => `TAG_TEMPLATE_APP_ID`

Reply:
Please take a look at the comments I've just added. I still believe `TAG_TEMPLATE_ID_APP` fits better because it means the _The ID of the Tag Template created to store additional metadata for App-related Entries_. WDYT?

Reply:
Got it, makes sense.